dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/TaskGroupServiceImpl.java

```diff
@@ -82,14 +82,13 @@
         if (groupSize <= 0) {
             putMsg(result, Status.TASK_GROUP_SIZE_ERROR);
             return result;
-
         }
         TaskGroup taskGroup1 = taskGroupMapper.queryByName(loginUser.getId(), name);
         if (taskGroup1 != null) {
             putMsg(result, Status.TASK_GROUP_NAME_EXSIT);
             return result;
         }
-        TaskGroup taskGroup = new TaskGroup(name, description,
+        TaskGroup taskGroup = new TaskGroup(name, projectcode,description,
                 groupSize, loginUser.getId(),Flag.YES.getCode());
         int insert = taskGroupMapper.insert(taskGroup);
```

Comment:
Please add a judgement  to  the variable of 'insert'. Seems like 'if(taskGroupMapper.insert(taskGroup) >0){...} else {...}'. You can refer to a few counterparts in other service implements.

Reply:
I will fix it